### PR TITLE
CVideoPlayer: use std::chrono

### PR DESCRIPTION
**Pull request by lrusak**
More cleanup to go with #19608 but the changes here are more significant and worth a separate PR.

Tested with a simple play test of a few files including a Bluray ISO.

### CODE REVIEW

**Comment by lrusak** [comment]
jenkins build this please

**Comment by lrusak** [comment]
Not sure who can have a look at this PR as a sanity check

File: xbmc/cores/VideoPlayer/VideoPlayer.cpp [hunk 1]
```diff
@@ -1446,12 +1446,15 @@ void CVideoPlayer::Process()
         // stills will be skipped
         if(m_dvd.state == DVDSTATE_STILL)
         {
-          if (m_dvd.iDVDStillTime > 0)
+          if (m_dvd.iDVDStillTime > 0ms)
           {
-            if ((XbmcThreads::SystemClockMillis() - m_dvd.iDVDStillStartTime) >= m_dvd.iDVDStillTime)
+            const auto now = std::chrono::steady_clock::now();
```
Comment (fuzzard) [inline xbmc/cores/VideoPlayer/VideoPlayer.cpp:1451]: #include <chrono>?

File: xbmc/cores/VideoPlayer/VideoPlayer.h [hunk 2]
```diff
@@ -514,15 +515,17 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
       iSelectedSPUStream   = -1;
       iSelectedAudioStream = -1;
       iSelectedVideoStream = -1;
-      iDVDStillTime        =  0;
-      iDVDStillStartTime   =  0;
+      iDVDStillTime = std::chrono::milliseconds::zero();
```
Comment (garbear) [inline xbmc/cores/VideoPlayer/VideoPlayer.h:518]: You use `std::chrono::milliseconds::zero()` here but `0ms` elsewhere?

Comment (lrusak) [reply xbmc/cores/VideoPlayer/VideoPlayer.h:518]: That's a good question. I believe the answer is that I don't want to include the `using namespace std::chrono_literals` in the header file.

**Review by AlwinEsch — APPROVED** [review]
From them what I see should it works correct and now more clean.

File: xbmc/cores/VideoPlayer/VideoPlayer.h (hunk 2 above)
Comment (garbear) [reply xbmc/cores/VideoPlayer/VideoPlayer.h:518]: fair

**Review by garbear — APPROVED** [review]

**Pull request merged**